### ISSUE TITLE
freelan windows tap adapter problem

### ISSUE THREAD

**Issue by richman1000000** [freelan-developers/freelan#223]
freelan still has this problem in windows.
After connection is established traffic is not going out or in.
If you open TAP adapter properties you can see that it set to Manual IP but  no IP address is specified.

all fields are empty.
![image](https://user-images.githubusercontent.com/9943374/83938021-06fa5680-a7da-11ea-9b06-c2fedd246187.png)
![image](https://user-images.githubusercontent.com/9943374/83937870-12994d80-a7d9-11ea-9583-d3cc11c524e7.png)

After setting it back to Automatically traffic starts to flow. Until service is restarted

**Comment by asafgb** [comment]
write here your config file

**Comment by richman1000000** [comment]
[freelan.txt](https://github.com/freelan-developers/freelan/files/6156847/freelan.txt)

**Comment by asafgb** [comment]
run the freelan.exe with parameter -d
steps
1. open cmd as admin
2. stop the service of freelan
3. cd the cmd to freelan Folder
4. run freelan.exe -d
5

So your config is that

> 

> [server]
> [client]
> public_endpoint=0.0.0.0
> 
> [fscp]
> contact=x.x.x.x:12000
> dynamic_contact_file=hw-sweethome.crt
> never_contact=10.17.2.38
> 
> [tap_adapter]
> mss_override=1300
> ipv4_address_prefix_length=10.17.2.38/28
> dhcp_server_ipv4_address_prefix_length=10.17.2.38/28
> [switch]
> [router]
> 
> [security]
> signature_certificate_file=hw-hp-p14.crt
> signature_private_key_file=hw-hp-p14.pem
> authority_certificate_file=CA.crt
> 
>

**Comment by richman1000000** [comment]
[freelan_run_debug.txt](https://github.com/freelan-developers/freelan/files/6157411/freelan_run_debug.txt)

**Comment by asafgb** [comment]
I new as you on that Software
but It looks on the debug that it success to reach
it reach to Destination computer
what the debug On the other computer

**Comment by richman1000000** [comment]
no, problem is always happens on the Windows. I was testing freelan on Windows 7 about 5 years ago. The issue still exists on Windows 10 and Windows 2019 server.
Currently I only have 1 windows computer - my laptop which runs freelan.
All others machines are linux, they have no problem.
